README.rst

```diff
@@ -933,32 +933,36 @@ Choose from the list of available rules:
   - ``only_untyped`` (``bool``): whether to add missing ``@param`` annotations for
     untyped parameters only; defaults to ``true``
 
-* **phpdoc_align** [@Symfony]
+* **phpdoc_align** [@Symfony, @Laravel]
 
   All items of the given phpdoc tags must be aligned vertically.
 
   Configuration options:
 
+  - ``parts`` (``array``): parts should be aligned; defaults to ``['hint', 'var',
```

Comment:
From the original regex: https://github.com/valentinxxx/PHP-CS-Fixer/blob/2.4/src/Fixer/Phpdoc/PhpdocAlignFixer.php#L63-L71

`<tag>`, `<hint>`, `<var>`, `<desc>`